database/migrations/2016_03_22_091319_create_orders_details_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateOrdersDetailsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders_details', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('orders_id')->unsigned();
+            $table->integer('products_id')->unsigned();
+            $table->integer('qty_pro_ordered')->unsigned();
```

Comment:
why you named `qty_pro_ordered`, `price_pro_ordered` it's difficult to understand